src/components/screens/votingQueue/editor/index.js

```diff
@@ -0,0 +1,14 @@
+/* istanbul ignore file */
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import { getActiveTokenAccount } from '../../../../utils/account';
+import Editor from './editor';
+
+const mapStateToProps = state => ({
+  account: getActiveTokenAccount(state),
+  votes: state.voting,
+});
```

Comment:
I would like to keep it like that so that I don't have to mock the store in the tests for this component. The tests become so large, unreadable and unmanageable with so many mocks that we need to add.